docs/users/students.md

```diff
@@ -60,6 +60,29 @@ Students whose access to a course has ended by being part of ended cohort or who
 
 Inactive students can be reactivated by moving to a cohort that is not ended or reactivating a dropped out student from the _Actions_ tab.
 
+## Student Standing
+
+Pupilfirst allows you to reward students for positive actions and penalize them for negative actions. This is done through a feature called [Standing](/users/school_standing).
+
+![Student standing page](../assets/students/school_student_standing_page.png)
+
+The standing page contains the following information:
+
+1. **Current Standing:** This section displays the current standing of the student with a standing shield and standing name, along with the color of standing.
+
+2. **Standing Logs:** This section displays the standing logs of students. Each standing log is displayed as a card with the following information:
+
+   - The standing shield styled with color of the standing.
+   - The standing name styled with color of the standing.
+   - The reason for change in standing.
```

Comment:
We shouldn't imply that a change in standing is always required when adding entries to the log.

```suggestion
   - The details of the standing log entry.
```